src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -141,6 +141,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('block.xml');
         $loader->load('menu.xml');
         $loader->load('commands.xml');
+        $loader->load('actions.xml');
```

Comment:
action.xml

Reply:
Why use singular? We use Commands and services

Reply:
What about block and menu?

Reply:
Maybe those are faulty, or maybe there is an unspoken rule about when to use singular and when to use plural, all I know is that in Symfony, you have `services.yaml`. Either way, fixing this is out of scope of this PR.

Reply:
Symfony is refering to a `services.yaml`:
https://symfony.com/doc/current/service_container.html#service-container-services-load-example

So we should use plurals 👍